src/Util/UrlManager.php

```diff
@@ -154,7 +154,7 @@ public function getBaseUrl(): string
     {
         if ($this->baseUrl === null) {
             try {
-                $process = $this->shell->execute('php bin/magento config:show:store-url default');
+                $process = $this->shell->execute('php bin/magento config:show:default-url');
 
                 $this->baseUrl = $process->getOutput();
             } catch (ShellException $e) {
```

Comment:
Message in exception need to change to avoid confusion. 
e.g. "Can't fetch base URL using command config:show:default-url. Instead URL from routes will be used.